featuretools_tsfresh_primitives/primitives/fft_coefficient.py

```diff
@@ -40,3 +45,115 @@ def function(x):
             return list(fft_coefficient(x, param=param))[0][1]
 
         return function
+
+
+class ShortTermFftCoefficient(TransformPrimitive):
+    """Calculates the spectral centroid (mean), variance, skew, and kurtosis of
+    the absolute fourier transform spectrum of entries over a given window.
+
+    Description:
+        Given a list of numbers and a corresponding list of
+        datetimes, return the spectral centroid (mean), variance, skew, and kurtosis of
+        the absolute fourier transform spectrum of
+        the numeric values, starting at the row `gap` rows away from the current row and
+        looking backward over the specified time window
+        (by `window_length` and `gap`).
```

Comment:
This docstring needs to be updated as it still references the values calculated by `ShortTermFftAggregated`